wildcard/lockdown/tests/test_lockdown.py

```diff
@@ -78,3 +76,14 @@ def test_condition_should_allow_commit(self):
         self.browser.getControl(name='title').value = newtitle
         self.browser.getControl(name='form.button.save').click()
         self.assertTrue(newtitle in self.browser.contents)
+
+    def test_show_status_message(self):
+        message = u'This site is in read-only mode!!!'
+        self.activateCondition()
+        self.browser.open(self.portal_url)
+        self.assertFalse(message.encode('utf8') in self.browser.contents)
+
+        self._settings.status_message = message
+        transaction.commit()
```

Comment:
Just FYI, it is fine here, but always double check that when you add a ``transaction.commit`` on a test, that the layer being used is a functional one (as in this case), if not, breakage on other packages or randomly on the same package can be seen (as the commit breaks the integration tests isolation)